nestest/test_nestest.py

```diff
@@ -64,7 +68,13 @@ def main():
             print(f'Missmatch detected on line: {line}')
             print('\n'.join(diff))
 
-    sys.exit(0 if success else 1)
+    if args.min_matching_lines:
+        if line < args.min_matching_lines:
+            print(f'Expected at least: {args.min_matching_lines} successful lines')
+            sys.exit(0 if success else 1)
```

Comment:
The exits here look weird. This should probably always be sys.exit(1), the else branch should go, and the last one should be checking success?

Reply:
I left the else case so we can use the return code in the ci job